lxd/instance_exec.go

```diff
@@ -277,6 +279,13 @@ func (s *execWs) Do(ctx context.Context, op *operations.Operation) error {
 			cmdErr = nil
 		}
 
+		// If the exec operation was deleted, the exec websocket is closed and the caller is booted from the instance.
+		// In this case, make sure there is a non-zero exit code. The disconnection exit code is used here.
+		if ctx.Err() != nil && cmdResult == 0 {
```

Comment:
@markylaing i just tested removing this part and I dont see that its needed, please can it be removed.